src/utils/COMETAdapter.cpp

```diff
@@ -298,7 +299,7 @@ class TOPPCOMETAdapter :
       else if (mod.getTermSpecificity() == ResidueModification::N_TERM)
       {
         term_distance = 0;
-        nc_term = "2";
+        nc_term = "0";
```

Comment:
correct?
The sixth entry specifies which terminus the distance constraint is applied to:
0 = protein N-terminus
1 = protein C-terminus
2 = peptide N-terminus
3 = peptide C-terminus